components/invitation_modal/invitation_modal_confirm_step_row.jsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import EmailIcon from 'components/svg/mail_icon';
+import GuestBadge from 'components/widgets/badges/guest_badge';
+
+import {imageURLForUser, isGuest} from 'utils/utils.jsx';
+
+import './invitation_modal_confirm_step_row.scss';
+
+export default class InvitationModalConfirmStepRow extends React.Component {
+    static propTypes = {
+        invitation: PropTypes.object.isRequired,
+    }
+
+    formatUserName = (user) => {
+        let displayName = '@' + user.username;
+        if (user.first_name || user.last_name) {
+            displayName = displayName + ' - ' + user.first_name + ' ' + user.last_name;
+        }
+        if (user.nickname) {
+            displayName = displayName + ' (' + user.nickname + ')';
+        }
+        return displayName;
+    }
```

Comment:
Actually the naming is not very good here, probably we can use a better name for the function. Anyway, it is "Format User Name", not "Format Username", because we are using the user data to format the name of the user including first and last name, nickname and things like that. Probably would be less confusing something like `formatUserDisplayName`.

Reply:
Done!